leaf-ui/rappid-extensions/LinkInspector.js

```diff
@@ -208,151 +281,62 @@ var LinkInspector = Backbone.View.extend({
         return node.prop('.funcvalue/text') == 'NB';
     },
     updateActorLink: function() {
-        var source = this.cell.getSourceElement();
-        var target = this.cell.getTargetElement();
-        this.cell.prop("link-type", $("#actor-link").val());
-        this.cell.label(0 , {position: 0.5, attrs: {text: {text: this.cell.prop("link-type")}}});
+        var linktype = $("#actor-link").val();
+        this.model.get('link').set("linkType", linktype);
+        this.model.label(0 , {position: 0.5, attrs: {text: {text: linktype}}});
     },
     // Generates the select values based on begin value
     updateBeginEvolRelations: function() {
+        var link = this.model.get('link');
+        //Enable the end select by changing the selected into true, because it doesn't change back to false, then end select will always be enabled from now on
+        this.model.set('selected', true);
+
         $("#repeat-error").text("");
         var begin = $("#link-type-begin").val();
-        var end = $("#link-type-end").val();
-        //Enable the end select
-        if (begin == "no") {
-            this.setSelectValues('#link-type-end', 'Evolving');
-            $("#link-type-end").prop('disabled', false);
-            $("#link-type-end").css("background-color", "");
-        } else if(begin == "and" || begin == "or") {
-            this.setSelectValues('#link-type-end', 'A');
-            $("#link-type-end").prop('disabled', false);
-            $("#link-type-end").css("background-color","");
-            
-            //Saving this option
-            this.cell.prop("link-type", begin + "|" + end);
-            this.cell.attr({
-                '.connection': {stroke: '#000000', 'stroke-dasharray': '0 0'},
-                '.marker-source': {'d': '0'},
-                '.marker-target': {stroke: '#000000', "d": 'M 10 0 L 0 5 L 10 10 L 0 5 L 10 10 L 0 5 L 10 5 L 0 5'}
-            });
+        var end = link.get('postType');
+        link.set("linkType", begin); //modify the model
+        var option = "#link-type-end option[value= \"" + begin + "\"]"; //for hiding the already chosen value at the beginning later on in line 294
+        
+        //makes the words on the links into lower case
+        if (['AND', 'OR', 'NO'].includes(begin)){
+            begin = begin.toLowerCase();
```

Comment:
You can use toLowerCase() on any string even if it has characters, so it might be simpler to just directly convert to lower case